quran/models.py

```diff
@@ -0,0 +1,18 @@
+'''Models describing abstractions used at Quran app'''
+from django.db import migrations, models
+
+
+class Surah(models.Model):
+    '''General information about Surahs'''
+    name = models.CharField(max_length=20)
+    total_ayahs = models.PositiveIntegerField(default=0)
+
+
+class Ayah(models.Model):
+    '''General information about Ayah'''
+    ayah = models.PositiveIntegerField(default=0)
```

Comment:
Add an `id` field. How else are you going to sort ayahs, when a frontend will make API call to get some specific ayah?
But don't make it a primary key, since it is not unique.
Default value `0` also doesn't make any sense.

Reply:
I added ayah field as identifier of ayah number.